Shuttle-back/src/main/java/com/shuttle/user/unregistered/UnregisteredUserService.java

```diff
@@ -0,0 +1,32 @@
+package com.shuttle.user.unregistered;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.shuttle.ride.dto.CreateRideEstimationDTO;
+import com.shuttle.ride.dto.EstimationDTO;
+import com.shuttle.vehicle.vehicleType.IVehicleTypeRepository;
+import com.shuttle.vehicle.vehicleType.VehicleType;
+
+@Service
+public class UnregisteredUserService implements IUnregisteredUserService{
+
+	@Autowired
+	private IVehicleTypeRepository vehicleTypeRepository;
```

Comment:
Service instead of repository.

Reply:
Spring does not let you to inject services in eachother simultaneously